src/Microsoft.OData.Edm/Validation/ValidationRules.cs

```diff
@@ -627,20 +627,16 @@ public static class ValidationRules
                     IEdmSchemaType schemaType = structuredType as IEdmSchemaType;
                     if (schemaType != null)
                     {
-                        var properties = structuredType.Properties().ToList();
-                        if (properties.Count > 0)
+                        foreach (var property in structuredType.Properties())

```

Comment:
Can we use IEdmProperty instead of var

Reply:
Good catch -- thanks!

---
In reply to: [543543057](https://github.com/OData/odata.net/pull/1930#discussion_r543543057) [](ancestors = 543543057)